post-deployment-scripts/sqstrigger.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+aws lambda create-event-source-mapping --function-name client-report-storage-master --batch-size 1 --maximum-batching-window-in-seconds 300 --event-source-arn arn:aws:sqs:eu-west-2:484165963982:dev-report-delivery-queue
```

Comment:
event-source-arn must be passed as input variable to bash script, shouldn't h/c

Also report/add some success/failure response back from cli

Reply:
resolved